rpcwebsocket.go

```diff
@@ -669,6 +1012,73 @@ func (m *wsNotificationManager) notifyForTxOuts(ops map[wire.OutPoint]map[chan s
 	}
 }
 
+// notifyRelevantTxAccepted examines the inputs and outputs of the passed
+// transaction, notifying websocket clients of outputs spending to a watched
+// address and inputs spending a watched outpoint.  Any outputs paying to a
+// watched address result in the output being watched as well for future
+// notifications.
+func (m *wsNotificationManager) notifyRelevantTxAccepted(tx *btcutil.Tx,
+	clients map[chan struct{}]*wsClient) {
+
+	var clientsToNotify map[chan struct{}]*wsClient
+
+	msgTx := tx.MsgTx()
+	for q, c := range clients {
```

Comment:
ACK. I've fixed this issue in `subscribedClients()` and refactored `notifyRelevantTxAccepted` here to call that.